### ISSUE TITLE
.JPG extension not working

### ISSUE THREAD

**Issue by Artmorse** [Schrodinger-Hat/ImageGoNord-cli#20]
**Describe the bug**
Using a `*.JPG` file is not working.

**To Reproduce**
```bash
python3 src/cli.py --img='./input-folder/01.JPG'
```

**Expected behavior**
This kind of extension should be managed or a clear message should inform the user how it's not working (with the real reason ; see the screenshot below :arrow_down:).

**Screenshots**
![image](https://github.com/Schrodinger-Hat/ImageGoNord-cli/assets/24521660/5d66ea4a-7f7d-41ad-a757-af91c959c80e)

**Comment by Wabri** [comment]
You are right, the regex that manage that value is wrong https://github.com/Schrodinger-Hat/ImageGoNord-cli/blob/main/src/cli.py#L149 it should be something like this:

```
^([A-z]|[\/|\.|\-|\_|\s])*\.([A-z]{3}|[A-z]{4})$
```

After that change it will work.

Do you mind open a pr with this change?

**Comment by Wabri** [comment]
Closed with merge #21
